<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelRow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество строк</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelColumn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество столбцов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinRow">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinColumn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Понеслось!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actioninfo"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actioninfo">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
